frontend/src/Calendar.js

```diff
@@ -2,19 +2,108 @@ import React from 'react'
 import FullCalendar from '@fullcalendar/react'
 import dayGridPlugin from '@fullcalendar/daygrid'
 
+import moment from 'moment'
+import timeGridPlugin from '@fullcalendar/timegrid'
+
+import {fetchUserCalendar} from './script'
+
+// the conversion of one minute to 60000 milliseconds
+const MIN_TO_MS = 60000;
+
 export default class Calendar extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      fetchDone: false,
+      schedulesTimes: [],
+      scheduleIdx: 0     // Current schedule
+    }
+  }
+
+  componentDidMount(){
+    this.getAllSchedules()
+  }
+
+  async getAllSchedules() {
+    const schedules = await fetchUserCalendar()
+    for(const schedule of schedules) {
+      this.getMeetingTimes(schedule)
+    }
```

Comment:
Change not necessary, but JS gives you a more concise way to express this kind of for-loop:

```
schedules.forEach(this.getMeetingTimes);
```

Reply:
Got it. Thank you